src/components/popup/LogIn.js

```diff
@@ -1,42 +1,75 @@
 import React from 'react';
-import { Modal, ModalBody, Button } from 'reactstrap';
+import { Modal, ModalBody, Button, Form, FormGroup, Input, Label } from 'reactstrap';
 import { connect } from 'react-redux';
 import { login } from '../../actions/auth';
 
+class loginModal extends React.Component {
 
-const loginModal = ({ auth, hideModals, login }) => {
-  const handleAdminLogin = event => {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      errors: {}
+    };
```

Comment:
smaž celý konstruktor, a místo něho dej pouze state (každopádně otázkou je zda state potřebuješ, protože this.state.errors nikde nepoužíváš, v tom případě není třeba aby to byla class):
`
state = {
   errors: {}
}
`